minio/requirements.txt

```diff
@@ -0,0 +1,34 @@
+asttokens==2.0.8
```

Comment:
Please, do not include this file in this PR as it belongs to a different topic/feature